pandas/core/tools/datetimes.py

```diff
@@ -759,13 +760,20 @@ def to_datetime(
         <https://docs.python.org/3/library/datetime.html
         #strftime-and-strptime-behavior>`_ for more information on choices, though
         note that :const:`"%f"` will parse all the way up to nanoseconds.
+        You can also pass:
+
+        - "ISO8601", to parse any ISO8601 time string (not necessarily in exactly the
```

Comment:
Might be to good add a link to what a ISO8601 string is?

Reply:
yup, thanks! apologies, missed that from Joris' review